File: AndroidUDPClient/src/com/example/androidudpclient/ProcessID.java
package com.example.androidudpclient;


public class ProcessID {

    // notify neighbor that fib is desired (sent in interest packet)
    static public final String REQUEST_FIB = "ReturnFIB";

    // notify recipient that content in data packet is FIBData
    static public final String FIB_DATA = "FIBData";
}

File: NDNHealthCloud/processid.js


/**
 * File contains constant strings that are sent withing
 * packets to identify what processing should be invoked;
 * hence, ProcessID.
 */

exports.ProcessID =  {
	// notify neighbor that fib is desired (sent in interest packet)
    REQUEST_FIB : "ReturnFIB",

    // notify recipient that content in data packet is FIBData
    FIB_DATA : "FIBData"
};